Source/Scene/Cesium3DTile.js

```diff
@@ -118,8 +120,6 @@ define([
          */
         this.computedTransform = computedTransform;
 
-        this._transformDirty = true;
```

Comment:
Removed this since it was only used in point clouds and it was easy to get around.